EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/83132
+discudemy.com#?#section.card:has(> label:contains(/^Ads$/) ~ ins.adsbygoogle)
```

Comment:
I'd prefer to use `:upward` -> `discudemy.com#?#article > section.card > ins.adsbygoogle:upward(section.card)`

Reply:
`:has` is faster than `:upward`.

Reply:
> but upward may better due less checking section elements

@krystian3w [said](https://github.com/AdguardTeam/AdguardFilters/issues/83132#issuecomment-840610710) a good thing.
@wout-codes about your rule. You're using double ext-css with Regex that should be avoided if possible.

Reply:
I think it's more safe, but I will add yours.

Reply:
> I think it's more safe

This is true, but it is not necessary here.